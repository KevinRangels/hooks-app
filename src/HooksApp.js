import React from 'react'

export const HooksApp = () => {
    return (
        <div>
            <h1>HOLA</h1>
        </div>
    )
}
