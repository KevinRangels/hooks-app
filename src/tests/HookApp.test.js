import React from 'react'
import { shallow } from 'enzyme'
import { HookApp } from '../HooksApp'
import "@testing-library/jest-dom"

describe('Pruebas in <HookApp>', () => {
    test('Debe mostrarse correctamente', () => {
        const wrapper = shallow( <HookApp/> )
        expect(wrapper).toMatchSnapshot()     
    })
    
})
