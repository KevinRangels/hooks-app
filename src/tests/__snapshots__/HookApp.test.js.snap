// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas in <HookApp> Debe mostrarse correctamente 1`] = `
<div>
  <h1>
    HOLA
  </h1>
</div>
`;
